.github/workflows/pr.yml

```diff
@@ -58,18 +57,31 @@ jobs:
         # 64 bit int
         cmake ${SHARED_OPT} -DBUILD_INDEX64=on -B lapack/build64
         cmake --build lapack/build64 ${PARALLEL} --target install
+    - name: Check for changes to this domain
+      id: changes
+      uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36 # v3.0.2
+      with:
+        filters: |
+          src:
+          - 'src/${{ matrix.domain}}/**'
+          - 'tests/unit_tests/${{ matrix.domain }}/**'
+          - CMakeLists.txt
+          - '.github/workflows/**'
```

Comment:
I would also add `examples` and `include` directories.

Reply:
I agree. Thanks, @rscohn2 !

Reply:
I am looking for a more maintainable way to specify the files that will trigger a test.